### PR TITLE
Cpdp 433 playback bpr

### PR DESCRIPTION
**Pull request by AndyHWChung**

### CODE REVIEW

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/check_your_details.scala.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+ cyaChange: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.cya_change,
+ addressDisplay: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.address_display,
+ submitButton: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.submit_button
+)
+
+@(bpr: BusinessPartnerRecord)(implicit request:Request[_], messages:Messages, appConfig: ViewConfig)
+
+@title = @{messages("subscription.title")}
+@label = @{Html(s"""<span class="form-label-bold">${messages("subscription.confirmLabel")}</span>""")}
+@emailAddress = @{bpr.emailAddress.map(e => e)}
```
Comment (AndyHWChung) [inline app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/check_your_details.scala.html]: This map isn't doing anything. 
```scala
bpr.emailAddress.map(e => e)
``` 
is equivalent to just 
```
bpr.emailAddress
```
You might not need this val here

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/cya_row.scala.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+@(label: String, value: Html, changeButton: Option[Html])
+
+<div>
+  <dt class="cya-question">
+   @label
+  </dt>
+  <dd class="cya-answer">
+   @value
+  </dd>
+  <dd class="cya-change">
```
[the remark this answers is not in the document]
Comment (nubz) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/components/cya_row.scala.html]: good point - changed.

File: app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/check_your_details.scala.html (hunk 1 above)
Comment (nubz) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/check_your_details.scala.html]: you are right - it's not used at all!

Comment (nubz) [reply app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/subscription/check_your_details.scala.html]: removed in latest commit

**Pull request merged**